src/libraries/Common/tests/System/Net/WebSockets/WebSocketCreateTest.cs

```diff
@@ -40,6 +40,7 @@ public void CreateFromStream_ValidBufferSizes_CreatesWebSocket()
         [OuterLoop("Uses external servers")]
         [Theory]
         [MemberData(nameof(EchoServers))]
+        [PlatformSpecific(~TestPlatforms.Browser)] // System.Net.Sockets is not supported on this platform.
```

Comment:
Should this be converted  to a new `PlatformDetection`, since there are other tests disabled for the same reason too?

Reply:
Should this really be in this PR?  Maybe another PR?

Reply:
Sure